src/RSGButton.js

```diff
@@ -65,15 +65,15 @@ export default function RSGButton (props: Props): React.createElement {
     }
   }
 
-  let activeStylesContent: Object = props.activeStyles && activeStylesList[props.activeStyles]
+  let activeStylesContent: any = props.activeStyles && activeStylesList[props.activeStyles]

```

Comment:
Avoid any. Please.
Use object shaping of Flow to define what this is equal to. It looks like a boolean, because && is not used to merge, it used to determine whether the properties exist. Fix this please, it's invalid.